etc/docker/enterprise-gateway-demo/bootstrap-enterprise-gateway.sh

```diff
@@ -5,13 +5,12 @@
 
 
 CMD=${1:-"--help"}
-if [[ "$CMD" == "--help" ]];
-then
+if [[ "$CMD" == "--help" ]]; then
 	echo ""
 	echo "usage: docker run -it[d] --rm -h <container-hostname> -p 8888:8888 [-p 8088:8088 -p 8042:8042] <docker-opts> <docker-image> <command>"
 	echo ""
 	echo "where <command> is:"
-	echo "    --jovyan ... Invokes Enterprise Gateway as user 'jovyan' directly.  Useful for daemon behavior."
+	echo "    --elyra ... Invokes Enterprise Gateway as user 'jovyan' directly.  Useful for daemon behavior."
```

Comment:
what if we name this --gateway ?

Reply:
That's fine - I'll make this change.  It will also affect the makefile.